ReactNativeNavigation.podspec

```diff
@@ -17,18 +17,61 @@ Pod::Spec.new do |s|
   
   s.subspec 'Core' do |ss|
     s.source              = { :git => "https://github.com/wix/react-native-navigation.git", :tag => "#{s.version}" }
-    s.source_files        = "lib/ios/**/*.{h,m,mm}"
+    # s.source_files        = "lib/ios/**/*.{h,m,mm}"
     s.exclude_files       = "lib/ios/ReactNativeNavigationTests/**/*.*", "lib/ios/OCMock/**/*.*"
   end
   
-  s.subspec 'Fabric' do |ss|
-    ss.xcconfig = { "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/RCT-Folly\"",
-                    "OTHER_CFLAGS" => "$(inherited) -DRN_FABRIC_ENABLED -DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1" }
-    ss.dependency 'React-RCTFabric'
-    ss.dependency 'React-Fabric'
-    ss.dependency 'RCT-Folly/Fabric'
+  reactJson = JSON.parse(File.read(File.join(__dir__, "node_modules", "react-native", "package.json")))
```

Comment:
I think this could fail in a monorepo, where `react-native` dependency could be located in other folder than _local `node_modules`_.
Anyway, it would be very unlikely since hoisting `react-native` bring too many problems.

Just wondering if there is an easy way to provide a fallback in case that happens (e.g. specifying the RN version manually).

What do you think @yogevbd?

Reply:
out of curiosity is this comment holding up the PR? perhaps the fallback can be about setting up an env var?

Just wondering was pending to push this forward ;)